app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -270,6 +270,34 @@ function handleKeyUp(event) {
   handleSizeChange(event);
 }
 
+function handleKeyDown(event) {
+    // If Ctrl+Enter OR Command+Enter is pressed
+    if((event.ctrlKey || event.metaKey) && event.keyCode === 13)
+    {
+        // Get user details and extract code of conduct & comment count
+        var user = userData();
+        if (!user){
```

Comment:
We should add a [space before the bracket](https://github.com/airbnb/javascript#whitespace--before-blocks) `{` here.